src/docs/common/snippets/common-nativetest.adoc

```diff
@@ -0,0 +1,71 @@
+:exclude-for-languages:groovy
+
+== Native Tests
+
+:exclude-for-build:maven
```

Comment:
should add maven snippet

Reply:
In this case, users need what is specified here: https://github.com/micronaut-projects/micronaut-platform/issues/934

Reply:
Actually never mind, Sergio actually wrote it